validation/gwas/method/regenie/README.md

```diff
@@ -0,0 +1,29 @@
+## REGENIE Validation
+
+The scripts in this directory are used to generate data and validate results from a reference implementation, specifically [GloWGR](https://glow.readthedocs.io/en/latest/tertiary/whole-genome-regression.html).  
+
+The general flow for this process is:
+
+1. Generate simulated genotypes, covariates, and traits saved as PLINK (via Hail) and pandas DataFrames
+2. Convert PLINK results to Zarr
+3. Run Glow WGR to produce results to compare against
+4. Export a subset of these results and the configuration that defines them to a unit test directory
+
+*Note*: The initial PLINK output is used for compatibility with the REGENIE C++ application
+
+All of the above are represented as pyinvoke tasks in [tasks.py](tasks.py).  
+
+The definition of each simulated dataset and parameterizations run against them can be seen in [config.yml](config.yml). 
+
+At TOW, these commands were used to generate the current test data:
```

Comment:
Not sure "TOW" is standard - took me a while to work out it means "time of writing".

Reply:
https://github.com/pystatgen/sgkit/pull/66/commits/3ddbd78756380368cfcbada1e247aa42737eb0ee#diff-89a72aaba83b82cd469b18bd9f23e3d0R18